configs/rec/crnn/crnn_icdar15.yaml

```diff
@@ -27,6 +27,7 @@ model:
   head:
     name: CTCHead 
     out_channels: *num_classes 
+  pretrained_ckpt: https://download.mindspore.cn/toolkits/mindocr/crnn/crnn_vgg7-ea7e996c.ckpt # support local path and url
```

Comment:
I think `pretrained` parameter should be versatile, i.e. able to accept `bool`, `str`, `url`, etc. For now, we have many different arguments that don't match each other and look confusing. For example, some of them: `pretrained`, `ckpt_path`, `checkpoint_path`, and now `pretrained_ckpt`.

I think we should stick to one `pretrained` parameter, because eventually it doesn't matter whether it's pretrained on ImageNet, SynthText, or local weights, the purpose of it remains the same.

Reply:
I see. But we cannot map the customized architecture defined in yaml to a pretrained model if we use `pretrained` and accept `bool` input.  
Indeed, it can be confusing for `pretrained` (for backbone only), `pretrained_ckpt`, and `ckpt_load_path` (for eval only) in one yaml file. I think renaming `pretrained_ckpt`  to 
 `initialize_from` will help alleviate it?.

Reply:
For the custom architecture scenario, when a user sets `pretrained` to `True` we can issue a warning saying that the pretrained model was not found and asking a user to specify a path to the pretrained model instead. Then, continue model initialization without loading pretrained weights.